shared/desktop/app/index.js

```diff
@@ -83,6 +84,11 @@ function start() {
     })
   })
 
+  ipcMain.on('win-service-check', (event, arg) => {
```

Comment:
or this should be more generic like 'startService' and on non-windows systems it effectively does nothing, that might be better